conda/exceptions.py

```diff
@@ -991,8 +991,8 @@ def __init__(self, caused_by, **kwargs):
         If you still have a need for Python 2 environments, consider using 'conda create'
         and 'conda activate'.  For example:
 
-            $ conda create -n py2 python=2
-            $ conda activate py2
+            $ conda create -n py3 python=3
+            $ conda activate py3
```

Comment:
The above blurb is explicitly discussing creating a Python 2 env so I'm not sure we want this change?

Reply:
Oh d'oh, you're right, jumping the gun!